libs/@guardian/source/package.json

```diff
@@ -40,6 +40,7 @@
 		"@babel/core": "7.24.0",
 		"@emotion/react": "11.11.1",
 		"@guardian/design-tokens": "workspace:*",
+		"@guardian/libs": "workspace:*",
```

Comment:
Should I use the workspace version of `libs`, or specify a specific version as in `source-react-components`?

Reply:
if we use the workspace version, it means needing to bump this package everytime `libs` changes, so i'd be inclined to not so that we can avoid that.

Reply:
I've updated to match `source-react-components`.